ourrevolution/settings.py

```diff
@@ -38,9 +38,20 @@
 
 
 # Application definition
-
+'''
+When several applications provide different versions of the same resource
```

Comment:
Figuring out this step was really confusing so I hope this is correct. I could not get admin template override to work until I did this part.

Reply:
If we override the template on a per model basis (https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#templates-which-may-be-overridden-per-app-or-model), we shouldn't need to change app order.

Reply:
My intention was to override for the whole admin theme, not a single model.

Reply:
Hmmmm. I'm fine with this. If we get complaints we can override per model. Disregard, but hey, at least you know all about overriding per model now haha.

Reply:
haha actually I figured out how to do it per model first, and then I spent extra time figuring out how to do it for the whole admin theme. So either way was a good learning experience 😄